README.md

```diff
@@ -0,0 +1,71 @@
+# Apache Beam (incubating) website
+
+This is the website for [Apache Beam](http://beam.incubator.apache.org)
+(incubating).
+
+### About this site
+The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
+for additional formatting capabilities, this website uses
+[Twitter Bootstrap](http://getbootstrap.com/).
+
+This website is presently hosted at:
```

Comment:
Though, now that I think about it more, we will probably want to fork this repo and create a new one for if we graduate from the Incubator. Fixed!